src/main/scala/com/fulcrumgenomics/fasta/CollectAlternateContigNames.scala

```diff
@@ -54,19 +54,19 @@ object AssemblyReportColumn extends FgBioEnum[AssemblyReportColumn] {
   }
 
   case object SequenceName     extends AssemblyReportColumn { val key: String = "Sequence-Name"; val tag: String = "sn" }
-  case object AssignedMolecule extends AssemblyReportColumn { val key: String = "Assigned-Molecule"; val tag: String = "am"  }
+  case object AssignedMolecule extends AssemblyReportColumn { val key: String = "Assigned-Molecule"; val tag: String = "am" }
   case object GenBankAccession extends AssemblyReportColumn { val key: String = "GenBank-Accn"; val tag: String = "ga"  }
   case object RefSeqAccession  extends AssemblyReportColumn { val key: String = "RefSeq-Accn"; val tag: String = "ra"  }
   case object UcscName         extends AssemblyReportColumn { val key: String = "UCSC-style-name"; val tag: String = "un"  }
 
   /** The missing value for a column */
-  val MissingValue: String = "na"
+  val MissingValueKey: String = "na"
 
   /** The column key for Sequence-Role */
-  val SequenceRole: String = "Sequence-Role"
+  val SequenceRoleKey: String = "Sequence-Role"
```

Comment:
Why not just make this a fully fledge `AssemblyReportColumn`?  You define it's tag below too.